compiler/src/yul/operations/calls.rs

```diff
@@ -13,3 +16,39 @@ pub fn contract_call(
     let func_name = names::contract_call(&contract.name, &func_name);
     expression! { [func_name]([address], [params...]) }
 }
+
+pub fn struct_new_call(struct_type: &Struct, params: Vec<yul::Expression>) -> yul::Expression {
```

Comment:
The name of this module was a reference to the call opcode, which was not very clear.

In my current branch working with contract types, I've renamed this file to `contract.rs` and plan on using it for operations on contract types.

I think it would make sense to move these functions into a new file called `struct.rs`, similar to `contract.rs`, to indicate that this module contains operations on the struct type.

This method and the one below would could be changed to `new` and `get` or `get_attribute` then.